### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by lukashettler-isg**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 on:
-  push:    
+  push:
```
Comment (lukashettler-isg) [inline .github/workflows/main.yml:2]: @jurekseverin-isg 

can you please undo this change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,9 +15,17 @@ Types of changes
...
+- Added region folding [Pull #73](https://github.com/isg-stuttgart/vscode-isg-cnc/pull/73)
+- "Format Code" command now indents multilines
+- Autogenerate github pages at push to main branch
+- Filter option to either show grouped or line-by-line sorting of matches in sidebar [Pull #79](https://github.com/isg-stuttgart/vscode-isg-cnc/pull/79)
 
 ### Fixed
-
+- Sidebar cuts off all matches over 500 to prevent performance issues [Pull #74](https://github.com/isg-stuttgart/vscode-isg-cnc/pull/74)
+- Reinclude fixed docu feature [Issue #58](https://github.com/isg-stuttgart/vscode-isg-cnc/issues/58)
+- Add toLowerCase for nc file detection
```
Comment (lukashettler-isg) [inline CHANGELOG.md:26]: Can you please add the diff between the versions at the end of this file.

File: package.json [hunk 3]
```diff
@@ -52,8 +52,10 @@
         "onCommand:isg-cnc.FindNextTFS",
         "onCommand:isg-cnc.RemoveAllBlocknumbers",
         "onCommand:isg-cnc.AddBlocknumbers",
+        "onCommand:isg-cnc.StartDocu",
         "onCommand:isg-cnc.Beautify",
-        "onCommand:isg-cnc.FindNonAsciiCharacters"
+        "onCommand:isg-cnc.FindNonAsciiCharacters",
+        "onCommand:cnc-show-filecontent"
```
Comment (lukashettler-isg) [inline package.json:58]: what does the command "cnc-show-filecontent"?

File: src/extension.ts [hunk 4]
```diff
@@ -58,7 +60,6 @@ const regExTechnology = new RegExp("([TFS])([0-9]+)");
 // Blocknumber regex
 const regExpBlocknumbers = new RegExp(/^((\s?)((\/)|(\/[1-9]{0,2}))*?(\s*?)N[0-9]*(\s?))/);
 const regExpLabels = new RegExp(/(\s?)N[0-9]*:{1}(\s?)|\[.*\]:{1}/);
-
```
Comment (lukashettler-isg) [inline src/extension.ts:61]: can you please remove this unnecessary line?

File: .github/workflows/main.yml (hunk 1 above)
Comment (jurekseverin-isg) [reply .github/workflows/main.yml:2]: Is the reason to have less "changed files" or why should it be removed? I would say removing trailing spaces is good though

File: package.json (hunk 3 above)
Comment (jurekseverin-isg) [reply package.json:58]: It's a leftover from experimenting i think. I removed it, it isn't used anymore

File: CHANGELOG.md (hunk 2 above)
Comment (jurekseverin-isg) [reply CHANGELOG.md:26]: Done

File: src/extension.ts (hunk 4 above)
Comment (jurekseverin-isg) [reply src/extension.ts:61]: Done

File: .github/workflows/main.yml (hunk 1 above)
Comment (Luces92) [reply .github/workflows/main.yml:2]: Yes that's the reason. I want to see only code changes and not the beautifying changes. But i think we should talk about a code style we want to use and then use a linter to ensure this code style. Or maybe you already a linter?

Comment (jurekseverin-isg) [reply .github/workflows/main.yml]: I removed this change. All I use by now is ES-Lint, but it doesn't complain about trailing spaces, maybe it is configurable.
